PID test is 7277
2 scripts will run
    --python python1.py-- is running with pid 7286, log file name is -output-python_python1-7277.log
    --python python2.py-- is running with pid 7293, log file name is -output-python_python2-7277.log
 
--python python1.py-- pid(7286) Finished correctly at 2019-02-05 02:06:35
The command line --python python2.py-- ends with error code 4 at 2019-02-05 02:06:40

-------------------------------------Summary-------------------------------------
--python python1.py-- Finished correctly at 2019-02-05 02:06:35
--python python2.py-- Finished with error code 4 at 2019-02-05 02:06:40
---------------------------------------------------------------------------------

Something was wrong with some scripts :(
Test FAILED!!
